developing-on-openshift/transferring-files/02-02-downloading-files-from-a-container/assignment.md

```diff
@@ -0,0 +1,276 @@
+
+```
+oc expose service/simplemessage
+```
+
+You'll get output similar to the following.
+
+```
+route.route.openshift.io/simplemessage exposed
+```
+
+----
+
+`Step 6:`  Run the following command to get the URL to the SimpleMessage application that you'll access in a web browser:
+
+
+```
+export APP_ROUTE=`oc get route simplemessage -n myproject -o jsonpath='{"http://"}{.spec.host}'` && echo $APP_ROUTE
+```
+
+You'll get output similar to the following:
+
+```
+http://simplemessage-myproject.crc-dzk9v-master-0.crc.ogrx0anbjp74.instruqt.io
+```
+
+Remember: The output shown above is special to this running instance of OpenShift. The URL you get when you run this track will be different.
+
+Copy the URL returned in the output above and then copy it into a browser. The URL will take you to the SimpleMessage application's web page as shown in the figure below.
+
+![Original Web Sites](../assets/original-web-output.png)
+
+----
+
+`Step 6:` Run the following `oc rsh` command to open an interactive command shell within the `pod` whose name is defined in the environment variable `POD`:
+
+```
+oc rsh $POD
+```
+
+You'll get output similar but not exactly like the following:
+
+`sh-4.2$`
+
+----
+
+`Step 7:`  Run the following command from within the interactive shell in order to see files in the application's directory:
+
+```
+ls -las
+```
+
+This will yield output similar to:
+
+```
+total 32
+ 0 drwxrwxr-x.  1 default root   152 Mar 29 15:08 .
+ 0 drwxrwxr-x.  1 default root    17 Sep 15  2021 ..
+ 0 drwx------.  3 default root    25 Mar 29 15:08 .config
+ 0 drwxr-xr-x.  4 default root    70 Mar 29 15:08 .npm
+ 0 drwxr-xr-x.  2 default root     6 Mar 29 15:08 .npm-global
+ 0 drwxrwxr-x.  1 default root    19 Sep 15  2021 .pki
+ 4 -rw-r--r--.  1 root    root   998 Mar 29 15:01 index.js
+ 4 -rw-r--r--.  1 root    root    16 Mar 28 22:57 message.txt
+ 4 drwxr-xr-x. 52 default root  4096 Mar 29 15:08 node_modules
+16 -rw-r--r--.  1 default root 14411 Mar 29 15:08 package-lock.json
+ 4 -rw-r--r--.  1 root    root   285 Mar 28 22:57 package.json
+```
+
+This is a Node.js application. Node.Js is an interpreted programming language. The source code is readily available for editing and immediate deployment, with no compilation required.
+
+
+----
+
+`Step 8:`  Run the following command to confirm the directory within the container where the files are stored:
+
+```
+pwd
+```
+
+You will see the following output:
+
+```
+/opt/app-root/src
+```
+
+You will use this directory path in upcoming steps and topics.
+
+----
+
+`Step 9:`  Run the following to exit the interactive shell and return to the local machine:
+
+```
+exit
+```
+
+Next you will use the `oc rsync` command to copy files from the container into the local machine.
```

Comment:
Perhaps mention that there's no need to run this command in this step just yet? I got confused and ran it lol

Reply:
Changed the language to this: In upcoming steps you will use the `oc rsync` command to copy files from the container into the local machine.